src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -230,16 +230,23 @@ default String getEffectiveAvatarUrl()
      * Represents the bit offsets used by Discord for public flags
      */
     enum Flags{
-        NONE(            0, "None"),
-        EMPLOYEE(        1, "Discord Employee"),
-        PARTNER(         2, "Discord Partner"),
-        HYPESQUAD(       4, "HypeSquad Events"),
-        BUG_HUNTER_1(    8, "Bug Hunter Level 1"),
-        BRAVERY(        64, "HypeSquad House Bravery"),
-        BRILLIANCE(    128, "HypeSquad House Brilliance"),
-        BALANCE(       256, "HypeSquad House Balance"),
-        SYSTEM(       4096, "System"),
-        BUG_HUNTER_2(16384, "Bug Hunter Level 2");
+        NONE(                      0, "None"),
```

Comment:
How would this be used?

Reply:
Renamed it to `UNKNOWN` with -1 as value (Placeholder).